app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -60,10 +62,7 @@ ListingController = (
     $scope.showApplicationOptions = !$scope.showApplicationOptions
 
   $scope.toggleTable = (table) ->
-    $scope["active#{table}Class"] = if $scope["active#{table}Class"] then '' else 'active'
-
-  $scope.isActiveTable = (table) ->
-    $scope["active#{table}Class"] == 'active'
```

Comment:
`isActiveTable` is still used in `_features.html.slim:29` https://github.com/Exygy/sf-dahlia-web/pull/891/files#diff-4bdfc1842a1371d26e8dbea081644fd1L29

Reply:
Fixed in https://github.com/Exygy/sf-dahlia-web/pull/891/commits/a63ca3a7f35bc6c72857be7e53b8964c53542ec7